### PR TITLE
Create Project Info Page

### PR DESCRIPTION
**Pull request by CodyRichter**
As part of issue #67 add landing page with information on project creation before user is brought to form. This page will display the user's email associated with the account, which can be used in the future to add a user to a project's member list.

### CODE REVIEW

File: app/views/projects/project_submit_info.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  <h1 class="page-title">Project Creation Information</h1>
+</div>
+
+<div class="card">
+  <div class="card-body">
+    In order to be elegible to be judged and entered to win prizes, you must fist submit your project on Dashboard. Before creating an official submission, review the following requirements:
+    <br>
+    <li> You may only submit one application per project.
+    <li> Every member of the project team must be included in the submission.
+    <li> ADD MORE IMPORTANT FACTS HERE...
```
Comment (abochnak) [inline app/views/projects/project_submit_info.html.erb]: I'm going to wait to commit this until we iron out the participant rules.

File: app/views/projects/project_submit_info.html.erb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<div class="page-header">
+  <h1 class="page-title">Project Creation Information</h1>
+</div>
+
+<div class="card">
+  <div class="card-body">
+    In order to be elegible to be judged and entered to win prizes, you must fist submit your project on Dashboard. Before creating an official submission, review the following requirements:
```
Comment (abochnak) [inline app/views/projects/project_submit_info.html.erb]: *typo with eligible
*typo with first

File: app/views/projects/project_submit_info.html.erb (hunk 1 above)
Comment (jwoglom) [reply app/views/projects/project_submit_info.html.erb]: This should be custom copy in the configuration repo.

**Comment by CodyRichter** [comment]
Finished project info page changes, ready to be reviewed and merged.

**Pull request merged**